src/modules/ethereum/account-abstraction/createTypedData.ts

```diff
@@ -1,8 +1,45 @@
+import type { BytesLike, ethers } from 'ethers';
 import type { types as zkSyncTypes } from 'zksync-ethers';
 import { EIP712Signer, utils as zkSyncUtils } from 'zksync-ethers';
 import { invariant } from 'src/shared/invariant';
 import { normalizeChainId } from 'src/shared/normalizeChainId';
 
+type Eip712SignInput = ReturnType<(typeof EIP712Signer)['getSignInput']>;
```

Comment:
What is better to use in var names: `Eip` or `EIP`?

Reply:
Always lowercasing the trailing letters in camelCase is a good practice I found recently :)